stable/prometheus-operator/values.yaml

```diff
@@ -355,6 +355,10 @@ grafana:
   #   configMap: certs-configmap
   #   readOnly: true
 
+  ## If true, create a serviceMonitor for grafana
+  ##
+  serviceMonitor:
+    selfMonitor: true
```

Comment:
Since this chart has a `ci` directory, you should also add this value there, as this is what is used during the CI runs

Reply:
Added, thanks.